lib/LedgerSMB/Budget.pm

```diff
@@ -246,7 +246,7 @@ sub from_input {
              $line->{amount} =   $input->{"credit_$rownum"}
                                - $input->{"debit_$rownum"};
              $line->{credit} = $line->{amount} if $line->{amount} > 0;
-             $line->{debit}  = $line->{amount} * -1 if $line->{amount} < 0;
+             $line->{debit}  = -$line->{amount} if $line->{amount} < 0;
```

Comment:
Well, I don't have problem with this pull request being refused and I splitting it up for you.  I don't know how that is done (the rejection).    In your place, I'd probably be lazy and want to deal with it just once.

 I do not yet have the view that a PR is the unit of change, rather I think of commits as the atomic tracking unit.  I have loads to learn about git.  So I don't understand how the bug tracking will be easier.  

It is disheartening to find that the critiques I that I took to satisfy are not generally accepted by the team.   I have been on the other side where I have felt "noone told me, I did not agree to that."  In that way, I'm just caught in the middle.

It is heartening that  these mostly mind numbingly boring changes, bring some scrutiny to crufty code; for instance: you picking up that a megabyte is the wrong size.